docs/faqs/faqs.mdx

```diff
@@ -131,16 +131,14 @@ your code while continuing to get code from Amplication.
 
 ## Can I contribute to the development of Amplication?
 
-We'd love to have you contribute! To find out how,read -
-
-[Contributing](https://docs.amplication.com/contributing/)
+We'd love to have you contribute! To find out how,read our [Contributing](https://docs.amplication.com/contributing/) guide.
 
 ---
 
 ## Can I work with GitLab (or other git providers) besides GitHub?
 
-Amplication currently supports integration only with GitHub, but we are planning
-to add support for more Git providers. See our [Product Roadmap](https://docs.amplication.com/about/roadmap/).
+Amplication currently supports integration only with [GitHub](/sync-with-github) and [Bitbucket](/sync-with-bitbucket), 
```

Comment:
@dericksozo I think we need to remove the word "only".

@GreenMachine01 do you want to mention AWS Code Commit as well?

Reply:
I did ask to mention it. I think its part of another PR.